### DISCUSSION TITLE
Extracting Image EXIF Information on Upload

### DISCUSSION THREAD

**Discussion by Phil89a** [GetPublii/Publii#1497] [Help]
Newbie question here.   I am looking at using Publii to replace WordPress for my Photographic Gallery. What I can't find is any way of pre-populating the image details when uploading to a Gallery from the IPTC tags.  Is this possible ?

I have a lot of images and it would be too time consuming to manually fill these in :-(

**Comment by dziudek** [comment]
Hi,

Publii does not support such feature, and at this moment we have no plans to implement such feature as we have a long queue of much more often asked features

**Reply by michaelfluegge** [reply]
@dziudek , as this feature is quite common in WordPress galleries, as far as I know, I would like to know on which position in the queue this feature is today? 
It would would be cool to show not only the description of an image but also the EXIF data like camera, lens etc ( at least for me :-) )
